apps/frontend/src/components/global/ExportSplunkModal.vue

```diff
@@ -23,7 +23,7 @@
           <v-stepper-items>
             <v-stepper-content step="1">
               <AuthStep
-                index-to-show="hdf"
+                index-to-show=""
```

Comment:
add a note into the help area that the index needs to exist on the splunk instance before exporting to it